package.json

```diff
@@ -32,6 +33,9 @@
     "@types/node": "^4.0.47",
     "ts-node": "7.0.0",
     "tslint": "5.11.0",
-    "typescript": "3.0.1"
+    "typescript": "3.0.1",
```

Comment:
Maybe also add lodash types? https://www.npmjs.com/package/@types/lodash

Reply:
We were planning on adding them when we start using them. Will add them now.

Reply:
it's on line 32